__tests__/end-to-end/01login.test.js

```diff
@@ -46,7 +46,7 @@ describe(requirement(2), () => {
   });
 });
 
-describe(requirement(3), () => {
+describe.skip(requirement(3), () => {
```

Comment:
Oiie, acho que esses subiram juntos, mas é bom tirar pois o Renato falou que pode interferir no avaliador do github
```suggestion
describe(requirement(3), () => {
```